<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChartOptionsDialog</class>
 <widget class="QDialog" name="ChartOptionsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>469</width>
    <height>419</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New Chart...</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources.qrc">
    <normaloff>:/icons/chart-edit</normaloff>:/icons/chart-edit</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Graphs</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="2" column="2">
       <widget class="QCheckBox" name="cbGustWindSpeed">
        <property name="text">
         <string>Gust</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QCheckBox" name="cbPressure">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Barometric Pressure</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Humidity:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="cbTemperature">
        <property name="text">
         <string>Temperature</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QCheckBox" name="cbAverageWindSpeed">
        <property name="text">
         <string>Average</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Wind:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QCheckBox" name="cbWindDirection">
        <property name="text">
         <string>Direction</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QCheckBox" name="cbDewPoint">
        <property name="text">
         <string>Dew Point</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Other:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="cbIndoorTemperature">
        <property name="text">
         <string>Inside Temperature</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QCheckBox" name="cbHumidity">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Humidity</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QCheckBox" name="cbWindChill">
        <property name="text">
         <string>Wind Chill</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QCheckBox" name="cbIndoorHumidity">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Inside Humidity</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QCheckBox" name="cbApparentTemperature">
        <property name="text">
         <string>Apparent Temperature</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Temperature:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QCheckBox" name="cbRainfall">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Rainfall</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="lblSolar">
        <property name="text">
         <string>Solar:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QCheckBox" name="cbUVIndex">
        <property name="text">
         <string>UV Index</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QCheckBox" name="cbSolarRadiation">
        <property name="text">
         <string>Solar Radiation</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Period</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="TimespanWidget" name="timespan" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QGroupBox" name="gbAggregate">
     <property name="title">
      <string>Aggregate</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>This allows you to plot hourly averages, daily rainfall totals, monthly highs, etc.</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="AggregateOptionsWidget" name="aggregateWidget" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TimespanWidget</class>
   <extends>QWidget</extends>
   <header>timespanwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>AggregateOptionsWidget</class>
   <extends>QWidget</extends>
   <header>aggregateoptionswidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>cbTemperature</tabstop>
  <tabstop>cbIndoorTemperature</tabstop>
  <tabstop>cbApparentTemperature</tabstop>
  <tabstop>cbWindChill</tabstop>
  <tabstop>cbDewPoint</tabstop>
  <tabstop>cbHumidity</tabstop>
  <tabstop>cbIndoorHumidity</tabstop>
  <tabstop>cbUVIndex</tabstop>
  <tabstop>cbSolarRadiation</tabstop>
  <tabstop>cbAverageWindSpeed</tabstop>
  <tabstop>cbGustWindSpeed</tabstop>
  <tabstop>cbWindDirection</tabstop>
  <tabstop>cbPressure</tabstop>
  <tabstop>cbRainfall</tabstop>
  <tabstop>gbAggregate</tabstop>
 </tabstops>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ChartOptionsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
